app/javascript/src/admin/activities.js

```diff
@@ -445,6 +448,14 @@ function addCopyTableCallBack() {
   });
 }
 
+function copyPromoToClipboard() {
+  new Clipboard(".btn-clipboard-wapp", {
+    text: function (trigger) {
+      return trigger.getAttribute("data-clipboard-text");
+    },
+  });
+}
+
```

Comment:
Only interesting part of the `.js`-files, the rest was the linter adding some commas